src/mappings/ncit.sssom.tsv

```diff
@@ -9,7 +9,7 @@
 #   skos: http://www.w3.org/2004/02/skos/core#
 #   sssom: https://w3id.org/sssom/
 # license: https://w3id.org/sssom/license/unspecified
-# mapping_set_id: https://w3id.org/sssom/mappings/66648e7f-532b-4149-94fc-23c100d2699f
+# mapping_set_id: https://w3id.org/sssom/mappings/574a0bf3-f0df-4a09-aa25-df661df78c48
```

Comment:
## An aside on `mapping_set_id`
Should I create a low priority issue for this; to assign IDs to these, so that the randomly generated ones don't give us spurious diffs?

Reply:
I think these are fine, they give me some confidence that the pipeline is run, even when they are no changes

Reply:
I agree with Nico. The generated mapping_set_id values show evidence of the pipeline being run, similar to other files where the date is updated to the date of the pipeline run, and do not need a ticket to have anything changed here.